src/actions/index.js

```diff
@@ -22,6 +22,18 @@ export const connectToServer = ({ username, hostname, port }) => (dispatch, getS
         contents,
       })
       handleWebsocketsConnection(dispatch, getState)
+
+      AsyncStorage.getItem(`${username}-favorites`).then(favoritesString => {
+        try {
+          const favorites = JSON.parse(favoritesString)
+          dispatch({
+            type: types.FAVORITES_LOADED,
+            favorites,
```

Comment:
Small nitpick, what's the use of creating a variable here if we're already inside a try/catch?